roles/engineer.md

```diff
@@ -12,37 +12,14 @@ We need someone:
 
 We expect someone in the role:
 
+- to be billable for at least half their time after 2 weeks on-boarding
```

Comment:
Historically the half-time billable expectations was for less experienced joiner
In my head, there's an expectation of being closer to fully billable pre- achieving the +lead trait